reana_client/cli/workflow.py

```diff
@@ -196,10 +201,11 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
     help="""
     Start previously created workflow.
 
-    The workflow execution can be further influenced by setting operational
-    prameters using `-p` or `--parameter` option.  The option can be
-    repetitive. For example, to disable caching for the Serial workflow
-    engine, you can set ``-p CACHE=off``.
+    The workflow execution can be further influenced by setting input prameters
+    using `-p` or `--parameters` flag or by setting operational parameters
+    using `-o` or `--options`.  The input and operational parameters can be
```

Comment:
Better call `-o` consistently "options" or "operational options" rather than "operational parameters" in docstrings here and elsewhere.  I.e. reserve the word "parameters" for the `-p` only. In this way the people will more easily distinguish, they could even use informally "workflow parameters" and "workflow options" if they want...